test/handlers/ExtensibleFallbackHandler.spec.ts

```diff
@@ -628,6 +647,16 @@ describe("ExtensibleFallbackHandler", () => {
                 const { validator } = await setupTests();
                 expect(await validator.supportsInterface.staticCall("0x01ffc9a7")).to.be.true;
             });
+
+            it("should use less than 30.000 gas", async () => {
+                const { validator, erc165Bencher } = await setupTests();
+
+                for (const interfaceId of ["0x01ffc9a7", "0xdeadbeef"]) {
+                    const [cold, warm] = await erc165Bencher.supportsInterfaceGas(await validator.getAddress(), interfaceId);
+                    expect(cold).to.be.lessThan(30000);
```

Comment:
Question: Will there be a situation where `cold` gas usage is less than `warm` gas usage?